src/client/app/components/search/resultDetail.js

```diff
@@ -1,17 +1,29 @@
 import React, { PropTypes } from 'react';
 
-const ResultDetail = ({artist, album, releaseDate}) => {
-
+const ResultDetail = ({ artist, album, releaseDate }) => {
```

Comment:
I'm going to merge the PR but it's worth pointing out that since the style object doesn't need to be recreated every time the component is (which happens any time the props passed to the component change, so fairly often), so it would ideally be moved above to the module scope.